plugins/web/opentelemetry-instrumentation-web-exception/README.md

```diff
@@ -0,0 +1,100 @@
+# OpenTelemetry Web Exception Instrumentation
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+This module provides automatic instrumentation for capturing unhandled exceptions and promise rejections in web applications.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/instrumentation-web-exception
+```
+
+## Usage
+
+```typescript
+import { LoggerProvider } from '@opentelemetry/sdk-logs';
+import { EventLoggerProvider, events } from '@opentelemetry/api-events';
```

Comment:
The API has just the interface for EventLoggerProvider. Import the class from the Events SDK.